eclass/distutils-r1.eclass

```diff
@@ -1833,15 +1833,14 @@ distutils-r1_run_phase() {
 		# cythonize() but it's better than nothing)
 		local -x CYTHON_FORCE_REGEN=1
 
+		local x
+		for x in $(all-flag-vars); do
```

Comment:
I wonder if there's every a case for using `meson-python` without `DISTUTILS_EXT` — because right we have disjoint conditions. Perhaps we should add a `die` to `meson-python`  case below for the time being.

Reply:
meson-python is, for example, the build backend used by meson-python.

DISTUTILS_USE_PEP517=standalone which I suppose is different from here... also, very specifically, using it without DISTUTILS_EXT means you're using it for, well, pure `*.py` projects, thus we can reasonably conclude that you don't need all-flag-vars to be modified.

Reply:
Yeah, I don't think we need to worry about this for now. Maybe an `eqawarn` at best to ask people to tell us about the ebuild so we can poke at it and see if we need to do anything better in the eclass?

Reply:
Please just pick either and chuck it in, I can't get to it today.